mypy/types.py

```diff
@@ -2415,6 +2415,12 @@ def __init__(
         implicit: bool = False,
     ) -> None:
         super().__init__(line, column)
+        if fallback.type and fallback.type.fullname == "builtins.tuple":
+            assert len(fallback.args) == 1
```

Comment:
```suggestion
        super().__init__(line, column)
        if fallback.type and fallback.type.fullname == "builtins.tuple":
            # We actively ignore type argument for plain tuple fallback, to avoid spurious errors,
            # partial_fallback should never be used directly anyway, only via typeops.tuple_fallback().
            assert len(fallback.args) == 1
```